app/services/container_dashboard_service.rb

```diff
@@ -229,11 +229,8 @@ def daily_network_metrics
   end
 
   def daily_provider_metrics
-    if @daily_metrics.blank?
-      @daily_metrics = VimPerformanceDaily.find_entries(:tz => @controller.current_user.get_timezone)
-                                          .where(:resource => (@ems || ManageIQ::Providers::ContainerManager.all))
-                                          .where('timestamp > ?', 30.days.ago.utc).order("timestamp")
-    end
-    @daily_metrics
+    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', nil, nil, @controller.current_user.get_timezone)
```

Comment:
Every call but 2 use all of the args when calling `find_for_interval_name`. One skips `timestamp`, as we want to pass a non-range. This is the other instance, that skips these 2 args in the middle.

Using kwargs seems like over kill. On the other hand, 4 args seems a bit much as well. So if it increases readability, then sounds good to me.

@chessbyte were you thinking kwargs?